src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -49,6 +51,12 @@ public OI(Joystick driverStick, Joystick controlStick) {
         EJoystickButton backLeftButton = new EJoystickButton(driverStick, 5);
         backLeftButton.runWhileHeld(new AlignAndDistance(24));
 
+        EJoystickButton a = new EJoystickButton(driverStick, 1);
+        a.whileHeld(new Climb());
```

Comment:
I believe this will start a new command every tick, I think it would better if you used ```a.runWhileHeld(new Climb());